docs/3_2_model_exchange_api.adoc

```diff
@@ -239,13 +243,5 @@ _Please note: if all event types are given as `fmi3EventUnknown` then the functi
 Function <<fmi3GetDirectionalDerivative>>::
 See <<fmi3GetDirectionalDerivative>>.
 
-Note that simulation backward in time is only allowed over continuous time intervals.
-As soon as an event occurs (<<fmi3EnterEventMode>> was called), going back in time is forbidden, because <<fmi3EnterEventMode>> / <<fmi3UpdateDiscreteStates>> can only compute the next discrete state, not the previous one.
-
-Note that during <<InitializationMode>>, <<EventMode>>, and <<ContinuousTimeMode>>, <<input>> variables can be set with <<get-and-set-variable-values,`fmi3Set{VariableType}`>> and output variables can be retrieved with <<get-and-set-variable-values,`fmi3Get{VariableType}`>> interchangeably according to the model structure defined under element <<ModelStructure>> in the XML file.
```

Comment:
Is this explanation available somewhere else?

Reply:
Yes, 3.2. at the explanation of `fmi3SetTime`.

Reply:
@andreas-junghanns :  I do not find it.
I mean
 ```
Note that during <<InitializationMode>>, <<EventMode>>, and <<ContinuousTimeMode>>, <<input>> variables can be set with <<get-and-set-variable-values,`fmi3Set{VariableType}`>> and output variables can be retrieved with <<get-and-set-variable-values,`fmi3Get{VariableType}`>> interchangeably according to the model structure defined under element <<ModelStructure>> in the XML file.
```
and the example
```
_[For example, if one <<output>> `y1` depends on two <<input,`inputs`>> `u1`, `u2`, then these two <<input,`inputs`>> must be set, before `y1` can be retrieved._
_If additionally an <<output>> `y2` depends on an <<input>> `u3`, then `u3` can be set and `y2` can be retrieved afterwards._
_As a result, artificial or `real` algebraic loops over connected FMUs in any of these three modes can be handled by using appropriate numerical algorithms.]_
```

Where do I find this text or why is not needed any more?

Reply:
In the explanation for `fmi3SetTime` in file `3_2_model_exchange_api.adoc` line 74:
`_[This allows limited simulation backward in time._
_As soon as an event occurs (fmi3EnterEventMode was called), going back in time is impossible, because fmi3EnterEventMode/fmi3UpdateDiscreteStates can only compute the next discrete state, not the previous one.]_`

Reply:
I still do not get it.  The text passage you cite is clear and I find it. 
But what about the text passage and example that I cited here: https://github.com/modelica/fmi-standard/pull/1714#discussion_r825060523?

Reply:
Ah, I see. @TorstenBlochwitz and I agreed that this example is too obvious. We could move it to the IG. If you agree, please raise an issue there.